TestResultSummaryService/parsers/Parser.js

```diff
@@ -76,6 +76,31 @@ class Parser {
         return rerunLink;
     }
 
+    extractOpenjdkSha (output) {
+        let m;
+        let releaseInfo = '';
+        let openjdkSha = null;
+        const releaseInfoRegex = /=RELEASE INFO BEGIN=[\s\S]*?SOURCE="(.*?)"\n.*=RELEASE INFO END=/;
+        const hotspotOpenjdkRegex = /git:(.*)/;
+        const openJ9OpenjdkRegex = /OpenJDK:\s?([^\s\:]*)/;
+        if ( ( m = releaseInfoRegex.exec( output ) ) !== null ) {
+            releaseInfo = m[1];
+        }
+
+        if (releaseInfo.includes('git')) {

```

Comment:
- if `releaseInfo` is empty, we do not need to check. Further checks should be inside the above if block.
- use both `includes ` and regex seem redundant.